WcaOnRails/app/views/registrations/_register_form.html.erb

```diff
@@ -1,9 +1,10 @@
 <% disabled = !current_user.can_edit_registration?(@registration) %>
 <% url = @registration.new_record? ? competition_registrations_path(@competition)
                                    : registration_path(@registration) %>
+<% ineligible_events = @competition.competition_events.select{|e| !e.can_register?(current_user) }.map{|e| e.event } %>
```

Comment:
More nit: The lambda parameter in `select` should be called `ce` to avoid confusing `competition_event` and `event`.